Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:18-alpine
+WORKDIR /app
+COPY package*.json ./
```

Comment:
I think it's a good idea to explicitly copy the package.json and package-lock.json so that this will fail if someone removes the pacakge-lock.